packages/st2bundle/install_rhel7.md

```diff
@@ -0,0 +1,45 @@
+# Add EPEL repo
+sudo yum install -y epel-release
```

Comment:
then we should use ius as above then! https://ius.io/GettingStarted/#scripted-install